anilist.py

```diff
@@ -194,43 +195,29 @@ def match_to_plex(anilist_series: List[AnilistSeries], plex_series_watched: List
                     )
 
                     if not match:
+                        # Create first match dict for this anilist id
                         anilist_matches.append({
                             "anilist_id": matched_id,
-                            "watched_episodes": plex_season.watched_episodes,
-                            "total_episodes": plex_season.last_episode,
+                            "watched_episodes": mapped_start + plex_season.watched_episodes,
                             "mapped_seasons": [plex_season.season_number],
-                            "duplicate_mappings": 0,
                         })
                         continue
-                    # For multiple seasons with the same anilist id check how they should be added together
-                    elif (index != 0 and plex_season.first_episode > plex_seasons[index - 1].last_episode):
-                        match["watched_episodes"] = plex_season.watched_episodes
+                    # For multiple seasons with the same id
+                    # If the start of this season has been mapped use that
+                    if mapped_start:
+                        match["watched_episodes"] = mapped_start + plex_season.watched_episodes
                     else:
                         match["watched_episodes"] += plex_season.watched_episodes
 
-                        logger.warning(f"[MAPPING] Duplicate anilist-id found for custom mapping | "
-                                       f"title: {plex_title} | "
-                                       f"anilist id: {matched_id} | "
-                                       f"total watched episodes: {match['watched_episodes']} | "
-                                       f"previous season matches: {match['mapped_seasons']} | "
-                                       f"current season: {plex_season.season_number} |"
-                                       )
-
-                    match["total_episodes"] = + plex_season.last_episode
+                    # TODO support using number of last episode of the last season as a start
                     match["mapped_seasons"].append(plex_season.season_number)
```

Comment:
mapped_seasons is never read

Reply:
My itentention was to add it to the logger and present the List of seasons that matched with the same AnilistID.
Have added it to the logger instead of removing it, I think it would be beneficial for the users.